styles.css

```diff
@@ -0,0 +1,105 @@
+body {
+    background-color: #eee;
+}
+nav {
+  width: 100%;
+  height: auto;
+}
+.navbar-container {
+  float: right;
+  width: 300px;
+  display: flex;
+  text-align: right;
+}
+.navbar-button {
+  margin: auto;
+}
+.navbar-button button {
+  background-color: blue;
+  color: white;
+  border: 1px solid black;
+  padding: 10px;
+  border-radius: 8px;
+}
+header {
+  text-align: center;
+  padding-top: 80px;
+  clear: both;
+}
+#headerBrands {
+  width: 380px;
+  margin: auto;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 20px;
+}
+header img {
```

Comment:
Suggestion: Kindly don't use HTML tags directly, you can use classes instead.

Reply:
Hi, should we use this suggestion for every project or in this one in particular?

Reply:
For every project, [here](https://css-tricks.com/why-use-classes-or-ids-on-the-html-element/) is something worth reading on this topic.